infrastructure/tooling/src/smoketest/checks/createClient.js

```diff
@@ -0,0 +1,32 @@
+const taskcluster = require('taskcluster-client');
+
+exports.scopeExpression = {
+  AllOf: [
+    "auth:create-client:project/taskcluster/smoketest/*",
+    "auth:reset-access-token:project/taskcluster/smoketest/*",
+    "project:taskcluster:smoketest:*",
+  ],
+};
+
+exports.tasks = [];
+
+exports.tasks.push({
+  title: 'Create client and use it (--target client)',
+  requires: ['ping-auth'],
+  provides: [
+    'target-client',
+  ],
+  run: async () => {
+    const auth = new taskcluster.Auth(taskcluster.fromEnvVars());
```

Comment:
This `fromEnvVars` function is returning something of the shape `{rootUrl: ..., credentials: {clientId: .., accessToken: ..}}`.

Reply:
Oh, totally makes sense. Thank you!